src/deepsparse/benchmark/data_creation.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import glob
+import logging
+import random
+import string
+from typing import Dict, List, Tuple
+
+import numpy
+
+from deepsparse import Pipeline
+
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_STRING_LENGTH = 50
+DEFAULT_IMAGE_SHAPE = (240, 240, 3)
+
+__all__ = [
+    "get_input_schema_type",
+    "get_files_with_endings",
+    "generate_sentence",
+    "generate_image_data",
+    "load_image_data",
+    "generate_text_data",
+    "load_text_data",
+    "generate_question_data",
+    "load_question_data",
+]
+
+
+class SchemaType:
+    IMAGE: str = "images"
+    TEXT_SEQ: str = "sequences"
+    TEXT_INPUT: str = "inputs"
+    QUESTION: str = "question"
+
+
+def get_input_schema_type(pipeline: Pipeline) -> str:
```

Comment:
are all these functions just helper functions for the benchmarking pipeline or meant to be used as public functions, beyond benchmarking?

Reply:
I wrote them for the purpose of benchmarking, but since they're quite generic I thought they may be useful outside of the script. The `benchmark_pipeline.py` file was also getting super long, so I wanted to move some of the data functionality to a separate file